### PR TITLE
Upgrade to Reloaded.Memory 9.3.2

### PR DESCRIPTION
**Pull request by dreamsyntax**
* Remove unused imports
* Upgrade & Migrate to Reloaded.Memory 9.3.2
* Bump version to 1.1.1
* Upgrade test deps
* Tests target NET 8

### CODE REVIEW

File: AFSLib.Test/FileViewer.cs [hunk 1]
```diff
@@ -40,12 +39,12 @@ public void FileDatesAreGood()
...
-            Assert.Equal(2019, afsFileViewer.Metadata.Value[2].Year);
-            Assert.Equal(8, afsFileViewer.Metadata.Value[2].Month);
-            Assert.Equal(26, afsFileViewer.Metadata.Value[2].Day);
-            Assert.Equal(15, afsFileViewer.Metadata.Value[2].Hour);
-            Assert.Equal(12, afsFileViewer.Metadata.Value[2].Minute);
-            Assert.Equal(56, afsFileViewer.Metadata.Value[2].Second);
+            Assert.Equal(2019, afsFileViewer.Metadata.Value.Get(2).Year);
+            Assert.Equal(8, afsFileViewer.Metadata.Value.Get(2).Month);
+            Assert.Equal(26, afsFileViewer.Metadata.Value.Get(2)    .Day);
+            Assert.Equal(15,    afsFileViewer.Metadata.Value.Get(2) .Hour);
```
Comment (Sewer56) [inline AFSLib.Test/FileViewer.cs]: Too much space, 0/10

**Comment by Sewer56** [comment]
Run the tests, and I'll merge

**Comment by Sewer56** [comment]
force push, seems legit 10/10

**Comment by dreamsyntax** [comment]
> force push, seems legit 10/10

I have no idea what you are talking about.
I have deep fried the code.

**Pull request merged**